src/pages/documentation/media-for-edge-network/release-notes.md

```diff
@@ -0,0 +1,11 @@
+# Release Notes
+
+## June 6, 2023
+
+### iOS Edge Media v4.0.0
```

Comment:
- v4.0.0 -> 4.0.0 (respectively 2.0.0)
- remove double space before "This enables"
- release for -> release of the